examples/docker/docker-compose-full.yml

```
@@ -81,6 +81,7 @@ services:
       WASMCLOUD_ALLOW_FILE_LOAD: "true"
       WASMCLOUD_OCI_ALLOWED_INSECURE: registry:5000
       WASMCLOUD_OBSERVABILITY_ENABLED: "true"
+      OTEL_EXPORTER_OTLP_ENDPOINT: http://localhost:4318
```

```suggestion
      OTEL_EXPORTER_OTLP_ENDPOINT: http://otelcol:4318
```

Trying this locally, I found that the `otelcol` was the hostname that worked for me